youtube_dl/extractor/pluralsight.py

```diff
@@ -215,7 +219,7 @@ def _real_extract(self, url):
         for module in course_data:
             for clip in module.get('clips', []):
                 player_parameters = clip.get('playerParameters')
-                if not player_parameters:
+                if not player_parameters or clip.get('userMayViewClip') is False:
```

Comment:
i don't have an acount to test with.so if these can cause problems for registred users then just ignore it.
if raising an error if `authorized` is false in clip info is also doesn't work than i will close this PL and let an other one that have an account to fix this.